pkg/apis/rook.io/v1alpha2/types.go

```diff
@@ -27,12 +27,13 @@ import (
 // ************************************************************************************
 
 type StorageScopeSpec struct {
-	Nodes         []Node            `json:"nodes,omitempty"`
-	UseAllNodes   bool              `json:"useAllNodes,omitempty"`
-	TopologyAware bool              `json:"topologyAware,omitempty"`
-	NodeCount     int               `json:"nodeCount,omitempty"`
-	Location      string            `json:"location,omitempty"`
-	Config        map[string]string `json:"config"`
+	Nodes         []Node                 `json:"nodes,omitempty"`
+	UseAllNodes   bool                   `json:"useAllNodes,omitempty"`
+	TopologyAware bool                   `json:"topologyAware,omitempty"`
+	NodeCount     int                    `json:"nodeCount,omitempty"`
+	Location      string                 `json:"location,omitempty"`
+	Config        map[string]string      `json:"config"`
+	Storage       StorageClassDeviceSets `json:"storageClassDeviceSet,omitempty"`
```

Comment:
Do not name this `Storage`. Keep it as `StorageClassDeviceSets`. Similarly, the JSON name should be pluralized.

Reply:
done